package.json

```diff
@@ -1,28 +1,41 @@
 {
   "name": "pidusage",
-  "version": "2.0.3",
-  "description": "Cross-platform process cpu % and memory usage of a PID — Edit",
+  "version": "2.0.0",
+  "description": "Cross-platform process cpu % and memory usage of a PID",
+  "license": "MIT",
+  "homepage": "https://github.com/soyuka/pidusage",
+  "repository": "github:soyuka/pidusage",
+  "bugs": {
+    "url": "https://github.com/soyuka/pidusage/issues"
+  },
+  "author": "soyuka",
+  "contributors": [
+    "Simone Primarosa <simonepri@outlook.com> (https://simoneprimarosa.com)"
+  ],
   "main": "index.js",
+  "files": [
+    "lib",
+    "index.js"
+  ],
+  "engines": {
+    "node": ">=4"
+  },
+  "scripts": {
+    "lint": "standard index.js lib/*.js test/*.js",
+    "test": "npm run lint&& nyc ava -m \"!*benchmark*\"",
+    "bench": "ava -m \"*benchmark*\""
+  },
   "dependencies": {},
   "devDependencies": {
-    "faucet": "0.0.1",
+    "ava": "*",
```

Comment:
Fancy hipster test framework with 81 dependencies... 
Tape: not fancy, just works 13 dependencies...

Also why no version constraint?

Reply:
The output provided by ava is just far superior to tape.
Just watch the output from the CI.

In my opinion It makes the tests development much easier.

Also you can use ES6 syntax in tests also while testing on node versions that does not support it.